source/grid/grid_tools.cc

```diff
@@ -6330,100 +6326,232 @@ namespace GridTools
         process_answer,
         comm);
 
-      if (true)
+      result.sort_components_and_init_crs_structures(perform_handshake);
+
+      return result;
+    }
+
+
+
+    template <int dim, int spacedim>
+    DistributedComputePointLocationsInternal<dim, spacedim>
+    DistributedComputeIntersectionLocationsInternal<dim, spacedim>::
+      convert_to_distributed_compute_point_locations_internal(
+        const unsigned int                  n_quadrature_pnts,
+        const Triangulation<dim, spacedim> &tria,
+        const Mapping<dim, spacedim> &      mapping) const
+    {
+      if constexpr (dim > spacedim)
+        {
+          // compute_affine_transformation is not instantiated for dim >
+          // spacedim
+          AssertThrow(false,
+                      ExcMessage("Invalid combination of dim and spacedim ."));
+
+          (void)n_quadrature_pnts;
+          (void)tria;
+          (void)mapping;
+          return {};
+        }
+      else
```

Comment:
This cannot happen, can it!?

Reply:
[0fb0d20](https://github.com/dealii/dealii/pull/15165/commits/0fb0d2070fc05ecc30606c3f491308db4e6e60f7)